Sources/AblyAssetTrackingSubscriber/WorkerQueue/SubscriberWorkerQueueProperties.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import AblyAssetTrackingInternal
+
+internal class SubscriberWorkerQueueProperties: WorkerQueueProperties
```

Comment:
Same comments apply here as for `PublisherWorkerQueueProperties` re value semantics.

Reply:
fd7bf5c